Jenkinsfile

```
@@ -152,20 +152,22 @@ def release_candidate() {
           script: "git diff-index --name-only HEAD^ | grep -q TAG && " +
                   "grep -i '[0-9]rc[0-9]' TAG",
           returnStatus: true)) {
-        return false
+        return true
     }
     return false
 }
 
 def faults_enabled(String type) {
     // if the fault_enabled pragma is false or it a release candidate; disable fault injection
-    if ( ! cachedCommitPragma(pragma: 'faults_enabled', def_val: 'true') || (release_candidate())) {
+    if ((cachedCommitPragma(pragma: 'faults-enabled', def_val: 'true') != 'true') || release_candidate()) {
         return ""
     }
     if (type == "rpm") {
         return "--with fault-injection"
     }
-    return "--with-fault-injection"
+    if (type == "scons")
+        return "--with-fault-injection"
+    return ""
```

Might be better to have an `error` here as an unexpected value was passed:
```suggestion
    error("Unexpected value passed to faults_enabled(): " + type)
```